index.ts

```diff
@@ -132,7 +157,8 @@ async function main() {
           if (projectIssue.state === "closed") continue;
           // if issue doesn't have the "Price" label then skip it, no need to pollute repo with draft issues
           if (!(projectIssue.labels as GitHubLabel[]).some((label) => label.name.includes(LABELS.PRICE))) continue;
-
+          // if issue has the "Unavailable" label then skip it, no need to pollute repo with new issues that are already taken
+          if (!(projectIssue.labels as GitHubLabel[]).some((label) => label.name.includes(LABELS.UNAVAILABLE))) continue;
```

Comment:
Project (i.e. partner) issues don't have the `Unavailable` tag, only devpool issues from https://github.com/ubiquity/devpool-directory/issues

Here we should check that "if project issue is assigned to somebody then skip such issue", no need to check the `Unavailable` label